packages/playwright-core/src/client/network.ts

```diff
@@ -307,18 +307,31 @@ export class Route extends ChannelOwner<channels.RouteChannel> implements api.Ro
     this._reportHandled(true);
   }
 
-  async fulfill(options: { response?: api.APIResponse, status?: number, headers?: Headers, contentType?: string, body?: string | Buffer, path?: string } = {}) {
+  async fetch(options: FallbackOverrides = {}) {
+    this._checkNotHandled();
```

Comment:
Do we need this check? We might as well fetch after the route is handled?

Reply:
Sure, I can remove it.